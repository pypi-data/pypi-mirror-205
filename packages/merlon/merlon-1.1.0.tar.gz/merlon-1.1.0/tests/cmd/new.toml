bin.name = "merlon"
args = ["new", "Test mod"]
