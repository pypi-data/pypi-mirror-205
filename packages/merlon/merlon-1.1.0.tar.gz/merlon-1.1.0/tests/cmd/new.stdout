
Created package: Test mod by [..]
To build and run this package, run the following commands:

    cd "test-mod"
    merlon init
    merlon run

